.gitignore

```diff
@@ -5,6 +5,6 @@ output/*
 *.egg
 pep8_report.txt
 *vsdx
-
 # front-end dependencies
 visualization/components/
+.DS_STORE
```

Comment:
Maybe make this `*.DS_STORE` to cover the file if it appears in subfolders? (Not exactly sure if that's the right regex syntax.)